src/itemset.js

```diff
@@ -96,6 +97,12 @@ define([
                     return value;
                 },
                 validationFunc: function (mug) {
+                    if (!mug.options.lookupTablesEnabled) {
+                        return "You no longer have access to Lookup Tables in your application. " +
+                            "Lookup Tables are available on the Standard plan and higher.\n" +
+                            "Before you can make a new version of your application, " +
+                            "you must [change your subscription](" + changeSubscriptionLink + ") or delete this question.";
```

Comment:
This looks like it's going to dump the link text onto the page...can it be HTML? If not, it's probably better to have some hint text (something like "you must change your subscription (in Project Settings)") instead of the link.